interactive/models.py

```diff
@@ -18,8 +19,19 @@
 from .notifications import send_welcome_email
 
 
+def date_of_last_extract():
+    # The cutoff for TPP's data extract is the Wednesday of the previous week
+    # We usually receive the data on a Tuesday
```

Comment:
Could you expand this to "We usually receive the data on a Tuesday, so if today is a Tuesday or Wednesday, the cutoff is last Wednesday, otherwise it's the Wednesday before."?

Reply:
Fair point. Done.